apps/website/src/pages/admin/show-and-tell/review/[entryId]/index.tsx

```diff
@@ -154,6 +212,75 @@ const AdminReviewShowAndTellPage: NextPage<
           )}
         </Panel>
 
+        {entry && (
+          <>
+            <Headline>Mod comments:</Headline>
+            <Panel>
+              {entry.modComments.map((comment) => (
+                <div
+                  key={comment.id}
+                  className="flex flex-row items-center justify-between border-b py-2"
+                >
+                  <div>
+                    <span className="font-bold">{comment.user.name}</span> (
+                    <DateTime
+                      date={comment.createdAt}
+                      format={{ time: "minutes" }}
+                    />
+                    ): {comment.comment}
+                  </div>
+                  <div className="flex gap-2">
+                    <Button
+                      size="small"
+                      className={defaultButtonClasses}
+                      onClick={() =>
+                        toggleCommentVisibility(comment.id, comment.isInternal)
+                      }
+                    >
+                      {comment.isInternal ? (
+                        <>
+                          <IconEye className="h-6 w-6" /> Make Public
+                        </>
+                      ) : (
+                        <>
+                          <IconEyeSlash className="h-6 w-6" /> Make Private
+                        </>
+                      )}
+                    </Button>
+                    <Button
+                      size="small"
+                      className={dangerButtonClasses}
+                      confirmationMessage="Please confirm deletion!"
+                      onClick={() => deleteComment(comment.id)}
+                    >
+                      <IconTrash className="h-4 w-4" />
+                      Delete
+                    </Button>
+                  </div>
+                </div>
+              ))}
+
+              <form onSubmit={handleAddComment} className="mt-4 flex">
+                <input
+                  type="text"
+                  name="comment"
+                  placeholder="Add a comment..."
+                  className="flex-1 rounded-l border p-2 text-black"
+                  value={newComment}
+                  onChange={(e) => setNewComment(e.target.value)}
+                  required
+                />
+                <button
+                  type="submit"
+                  className="flex items-center justify-center rounded-r p-2 text-white"
+                >
+                  <IconPaperAirplane className="h-6 w-6" />
+                </button>
```

Comment:
Can this get a label? We usually give buttons a visual label even tho I like the icon. Also good for accessibility ;)